Scripts/Runtime/Jobs/AccessControl/AccessController.cs

```diff
@@ -0,0 +1,35 @@
+using System;
+
+namespace Anvil.Unity.DOTS.Jobs
+{
+    /// <summary>
+    /// A concrete implementation of <see cref="AbstractAccessController{T}"/> that assumes
+    /// wrapping a singular unchanging piece of data which is the most common case.
+    /// </summary>
+    /// <typeparam name="T">The type of data to wrap access control around.</typeparam>
+    public class AccessController<T> : AbstractAccessController<T>
```

Comment:
Proposal:
 - Remove `<T>` and value delivery from `AbstractAccessController`
 - Consider making `AccessController` non-abstract
 - Rename `AccessController<T>` to `AccessControlledValue<T>` and compose in the `AccessController`
 - For any more pieces of data that a developer wants to wrap into one control block they can either:
    - Wrap their data in an object and use `AccessControlledValue<T>`
    - Create their own class that composes in `AccessController`. With this option we'll just have to manually ensure the APIs stay as similar as possible between implementations. Docs should probably suggest that the `AccessControlledValue<T>` API is matched as closely as possible.

Reply:
As discussed, yes this is a good idea.

All changes made. 
https://github.com/decline-cookies/twisted-dots/commit/7aee552653a95a1fdc6ed857f863e30399636709 also updated to reflect the new changes.